src/test/java/net/imglib2/type/logic/BitTypeArrayImgSizeTest.java

```diff
@@ -0,0 +1,81 @@
+package net.imglib2.type.logic;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.Random;
+
+import net.imglib2.img.array.ArrayImg;
+import net.imglib2.img.array.ArrayImgFactory;
+import net.imglib2.img.array.ArrayImgs;
+import net.imglib2.img.basictypeaccess.array.ArrayDataAccess;
+import net.imglib2.img.basictypeaccess.array.LongArray;
+
+import org.junit.Test;
+
+public class BitTypeArrayImgSizeTest
+{
+
+	@Test
+	public void testOverSizedBitImageExplicitStorageArray()
+	{
+
+		final int numLongs = ( int ) Math.ceil( 1000 * 1000 / 64 );
+
+		final ArrayImg< BitType, LongArray > img = ArrayImgs.bits( new LongArray( new long[ numLongs ] ), 1000, 1000 );
```

Comment:
Would it make sense to also test with image sizes that are not divisible by 64? That is, `1000 * 1000 / 64 == 15625`, so we don't actually test "partially filled" `long`s. How about a 100 * 100 image (`1000 * 1000 / 64 == 156.25`)?

Reply:
Good point @imagejan I modified the test :+1: